src/index.html

```diff
@@ -4,9 +4,385 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700;900&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#index.html">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="logo"
+         >
+      </a>
+      <nav>
+        <ul class="nav-list">
+          <li class="nav-list__item">
+            <a href="#" class="nav-list__item-product is-active">Apple</a>
+          </li>
+          <li class="nav-list__item">
+            <a href="#" class="nav-list__item-product">Samsung</a>
+          </li>
+          <li class="nav-list__item">
+            <a href="#" class="nav-list__item-product">Смартфоны и телефоны</a>
+          </li>
+          <li class="nav-list__item">
+            <a
+            href="#"
+            data-qa="nav-hover"
+            class="nav-list__item-product"
+            >Ноутбуки и компьютеры</a>
+          </li>
+          <li class="nav-list__item">
+            <a href="#" class="nav-list__item-product">Гаджеты</a>
+          </li>
+          <li class="nav-list__item">
+            <a href="#" class="nav-list__item-product">Планшеты</a>
+          </li>
+          <li class="nav-list__item">
+            <a href="#" class="nav-list__item-product">Фото</a>
+          </li>
+          <li class="nav-list__item">
+            <a href="#" class="nav-list__item-product">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="container">
+        <div class="container__card-container">
+          <div class="card" data-qa="card">
+            <img
+              src="./images/imac.jpeg"
+              class="card__image"
+              alt="image_of_device"
+            >
+            <div class="card__info">
+              <p class="card__info-name">
+                Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+              </p>
+              <p class="card__info-code">
+                Код товара: 195434
+              </p>
+              <div class="card__info-review-wrapper">
+                <div class="card__info-stars stars stars--4">
+                  <div class="stars__star"></div>
+                  <div class="stars__star"></div>
+                  <div class="stars__star"></div>
+                  <div class="stars__star"></div>
+                  <div class="stars__star"></div>
+                </div>
```

Comment:
Check the first point from the [checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md)